cmd/talosctl/cmd/talos/processes.go

```diff
@@ -220,9 +220,11 @@ func processesOutput(ctx context.Context, c *client.Client) (output string, err
 				node = msg.Metadata.Hostname
 			}
 
+			fmt.Printf("`%s`\n", p.Label)
+
 			s = append(s,
-				fmt.Sprintf("%12s | %6d | %1s | %4d | %8.2f | %7s | %7s | %s",
-					node, p.Pid, p.State, p.Threads, p.CpuTime, humanize.Bytes(p.VirtualMemory), humanize.Bytes(p.ResidentMemory), args))
+				fmt.Sprintf("%12s | %6d | %1s | %4d | %8.2f | %7s | %7s | %64s | %s",
```

Comment:
64 is like a lot

Reply:
They're around 40 typically. The table shrinks anyway when column width isn't filled